### PR TITLE
IDE-3421 FileNotFoundException below PluginsSDKProjectProvider.createNewProject

### PR DESCRIPTION
**Pull request by simonjhy**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java [hunk 1]
```diff
@@ -227,6 +229,10 @@ public IStatus createNewProject(NewLiferayPluginProjectOp op, IProgressMonitor m
 
 			projectParent.mkdirs();
 
+			if (!FileUtil.exists(newSDKProjectPath)) {
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java]: use FileUtil.notExists instead

**Comment by jtydhr88** [comment]
see comments, thanks

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java [hunk 2]
```diff
@@ -227,6 +229,10 @@ public IStatus createNewProject(NewLiferayPluginProjectOp op, IProgressMonitor m
...
+				return ProjectCore.createErrorStatus("Error create project folder.");
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/PluginsSDKProjectProvider.java]: @Camork lets give more detail information to users

**Comment by jtydhr88** [comment]
@simonjhy @Camork see my comments and resend, thanks.

**Comment by Camork** [comment]
OK

**Pull request closed without merging**